test/common/router/retry_state_impl_test.cc

```diff
@@ -834,21 +834,21 @@ TEST_F(RouterRetryStateImplTest, Backoff) {
   setup(request_headers);
   EXPECT_TRUE(state_->enabled());
 
-  EXPECT_CALL(random_, random()).WillOnce(Return(49));
-  retry_timer_ = new Event::MockTimer(&dispatcher_);
-  EXPECT_CALL(*retry_timer_, enableTimer(std::chrono::milliseconds(24), _));
+  EXPECT_CALL(random_, random()).WillOnce(Return(190));
+  retry_timer_ = new testing::StrictMock<Event::MockTimer>(&dispatcher_);
```

Comment:
FYI we have StrictMock as default in our test runs, so this shouldn't be needed AFAIK

Reply:
Sorry, this was left over from earlier attempts to debug the test failures.